run-composed

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+# See https://github.com/kabisaict/jenkins-docker for more information.
+#
+# Usage:
+# run-composed [CI command]
+#
+# Examples:
+# run-composed 'bin/ci'
+# run-composed 'mvn clean install'
+set -o pipefail
+trap cleanupContainer EXIT
+
+function cleanupContainer() {
```

Comment:
Does this make sense for docker compose? Not that familiar with compose, but I guess you'd want to shutdown all containers linked to the compose instance?